### PR TITLE
Fixed crash with accents in page title

### PR DESCRIPTION
**Pull request by mwss1996**
The str() method gives me an encoding error when i try to use brazilian accentuation (á, ã, ê). Changing to unicode() corrected the problem.

### CODE REVIEW

**Comment by dpiquet** [comment]
Same problem here. This PR works for me

File: ckanext/pages/plugin.py [hunk 1]
```diff
@@ -48,9 +48,9 @@ def build_pages_nav_main(*args):
 
     for page in pages_list:
         if page['page_type'] == 'blog':
-            link = h.literal('<a href="/blog/%s">%s</a>' % (str(page['name']), str(page['title'])))
+            link = h.literal('<a href="/blog/%s">%s</a>' % (unicode(page['name']), unicode(page['title'])))
```
Comment (torfsen) [inline ckanext/pages/plugin.py]: I think the first value should be URL-escaped and the second needs to be HTML-escaped instead. AFAIK, `h.literal` by design does neither.

File: ckanext/pages/plugin.py [hunk 2]
```diff
@@ -48,9 +48,9 @@ def build_pages_nav_main(*args):
...
         else:
-            link = h.literal('<a href="/pages/%s">%s</a>' % (str(page['name']), str(page['title'])))
+            link = h.literal('<a href="/pages/%s">%s</a>' % (unicode(page['name']), unicode(page['title'])))
```
Comment (torfsen) [inline ckanext/pages/plugin.py]: Same issue here w.r.t. URL/HTML-escaping.

**Comment by torfsen** [comment]
Closed in favour of #64.

**Pull request closed without merging**